_episodes/01-variables.md

```diff
@@ -0,0 +1,356 @@
+ - cannot start with a digit;
+ - are case-sensitive.
+
+This means that, for example:
+ - `weight0` is a valid variable name, whereas `0weight` is not;
+ - `weight` and `Weight` are different variables.
+
+## Using Variables in Python
+
+Once we have data stored in variables, we can make use of it in calculations.
+We may be interested in our patient's weight in pounds:
+
+~~~
+# There are 2.2 pounds per kilogram
+2.2 * weight_kg
+~~~
+{: .language-python}
+~~~
+132.66
+~~~
+{: .output}
+
+We might decide to add a prefix to our patient identifier:
+
+~~~
+patient_id = 'inflam_' + patient_id
+~~~
+{: .language-python}
+
+Variables must be created before they are used.
+
+## Using Built-in Functions
+
+To carry out common tasks with data and variables in Python,
+the language provides us with several built-in functions.
+To display information to the screen, we use the `print` function:
+
+~~~
+print(weight_lb)
+print(patient_id)
+~~~
+{: .language-python}
+~~~
+132.66
+inflam_001
+~~~
+{: .output}
+
+When we want to make use of a function, which is referred to as _calling_ a
+function, we follow its name by parentheses. The parentheses are important:
+If you leave them off, the function doesn't actually run.
+Sometimes we will include values or variables inside the parentheses.
+In the case of `print`, we include what we want to display.
+
+~~~
+print("hello")
+print("3")
+print(3)
+~~~
+{: .language-python}
+~~~
+hello
+3
+3
+~~~
+{: .output}
+
+We will learn more about how functions work and how to create our own in a
+later episode.
+
+We can display multiple things at once using only one `print` call:
+
+~~~
+print(patient_id, 'weight in kilograms:', weight_kg)
+~~~
+{: .language-python}
+~~~
+inflam_001 weight in kilograms: 60.3
+~~~
+{: .output}
+
+We can also call a function inside of another function call.
+For example, Python has a built-in function called `type` that tells us a
+value's data type:
+
+~~~
+print(type(60.3))
+print(type(patient_id))
+~~~
+{: .language-python}
+
+~~~
+<class 'float'>
+<class 'str'>
+~~~
+{: .output}
+
+Another common built-in function is `len`: It is used to find the length of a
+character string:
+
+~~~
+print(len("hello"))
+print(len('weight in pounds:'))
+~~~
+{: .language-python}
+~~~
+5
+17
+~~~
+{: .output}
+
+Moreover, we can do arithmetic with variables right inside the `print` call:
+
+~~~
+print('weight in pounds:', 2.2 * weight_kg)
+~~~
+{: .language-python}
+~~~
+weight in pounds: 132.66
+~~~
+{: .output}
+
+## Changing the Value of Variables
+
+The above command did not change the value of `weight_kg`:
+
+~~~
+print(weight_kg)
+~~~
+{: .language-python}
+~~~
+60.3
+~~~
+{: .output}
+
+To change the value of the `weight_kg` variable, we have to _assign_
+`weight_kg` a new value using the equals `=` sign:
+
+~~~
+weight_kg = 65.0
+print('weight in kilograms is now:', weight_kg)
+~~~
+{: .language-python}
+~~~
+weight in kilograms is now: 65.0
+~~~
+{: .output}
+
+> ## Variables as Sticky Notes
```

Comment:
why to change the formatting from here on?

Reply:
Oh, it comes from: https://swcarpentry.github.io/python-novice-inflammation/01-intro/index.html#variables-as-sticky-notes "Variables as Sticky Notes" is a 'callout' section and the rest are challenges (see https://swcarpentry.github.io/python-novice-inflammation/01-intro/index.html#check-your-understanding and so on).

Basically, I left the original formatting... I think it doesn't hurt, but I should probably copy SWC's CSS files for this render as expected.

Reply:
great. For now it just displays as a shifted paragraph so I was confused :-)